dozer-types/src/arrow_types/from_arrow.rs

```diff
@@ -142,12 +145,31 @@ macro_rules! make_text {
     }};
 }
 
+macro_rules! make_json {
```

Comment:
I feel these macros are bad practice. What do you feel during writing code? Should we change it to a function?

Reply:
right - agree on changing to function - but then it will exactly the same way that we are doing now but in single function with `match` to handle different type cases

Reply:
I mean, to start with, we only change `make_json` to a function, leaving other parts untouched.

Reply:
sure